docs/autostarting.md

```diff
@@ -53,15 +53,14 @@ This file needs to be copied over to the supervisor configuration directory at `
 
 To enable NGINX and the supervisor, run the following commands (these also set the proper permissions):
 
-```
     sudo chgrp -R www-data /var/log/security_monkey
+    sudo chmod g+w /usr/local/src/security_monkey
```

Comment:
I suppose I'm a little hesitant on giving the app user direct write access to itself vs. a more scoped path.

Reply:
Certainly fair, I have gone back and forth a few times. /var/run is optimal, but dirs/permissions won't survive a reboot (same for /tmp if creating a subdir under there). 

Basically its this issue, https://stackoverflow.com/questions/31458842/how-to-create-a-directory-in-run-for-each-supervisor-program (where the choices seem to be an init script or a wrapper)